src/ventus/wm/window.js

```diff
@@ -487,6 +488,7 @@ function(Emitter, View, WindowTemplate) {
 
 		close: function() {
 			this.closed = true;
+			this.opened = false;
```

Comment:
@macqm This is something that I need to change, since I'm using the setters to update the window state we need to set both ```opened``` and ```closed```. It doesn't make sense to have two different attributes that actually represent the same value. 

The ```opened``` setter should set ```closed``` to false, and viceversa. I'll take the logic out of the setters.

Thanks!